app/code/core/Mage/Centinel/Block/Adminhtml/Validation.php

```diff
@@ -64,11 +64,9 @@ protected function _toHtml()
         if (!$payment->getMethod()
             || !$payment->getMethodInstance()
             || $payment->getMethodInstance()->getIsDummy()
-            || !$payment->getMethodInstance()->getIsCentinelValidationEnabled())
-        {
+            || !$payment->getMethodInstance()->getIsCentinelValidationEnabled()) {
```

Comment:
Not sure about this:
```suggestion
            || !$payment->getMethodInstance()->getIsCentinelValidationEnabled()
        ) {
```